app/models/organization_user.rb

```diff
@@ -4,6 +4,8 @@ class OrganizationUser < ApplicationRecord
   belongs_to :organization
   belongs_to :user
 
+  enum status: { invited: "invited", archived: "archived" }
```

Comment:
Shouldn't there be an `active` status? when the user accepted the invitation?

Reply:
Key and value aresame for enum. Why are defining it as hash

You could define it like this 

` enum status: [ :active, :invited, :archived ]`

Reply:
So I might be missing something but when using Rails enums it uses int values. 

https://rails.rubystyle.guide/#enums